recipes/ddtrace/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ddtrace" %}
+{% set version = "0.42.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ddtrace-{{ version }}.tar.gz
+  sha256: dba66c7013a4352dfa20ed31e989001250056c89811d1cb5ee10e23dc98bac0a
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py==34]
+  entry_points:
+    - ddtrace-run = ddtrace.commands.ddtrace_run:main
+    - pyddprofile = ddtrace.profiling.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python
+    - setuptools_scm
+    - setuptools_scm >=4
```

Comment:
You should only have one of the above two lines

Reply:
That makes sense, thank you